ModManagerUI/LocalizationSystem/LocalizationPatcher.cs

```diff
@@ -0,0 +1,36 @@
+﻿using HarmonyLib;
+using MonoMod.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ModManagerUI.LocalizationSystem
+{
+    public class LocalizationPatcher
+    {
+        public static void Patch(Harmony harmony)
+        {
+            //var harmony = new Harmony("TimberApi.Localization");
```

Comment:
Is this also in TimberAPI? or just because you made it a static?
Can be removed :D

Reply:
Yeah, the localizationsystem is copy-pate as it's finest :D the line is there in TimberAPI but uncommented :)